azure-quantum/azure/quantum/qiskit/results/resource_estimator.py

```diff
@@ -6,7 +6,63 @@
 To install run: pip install azure-quantum[qiskit]"
     )
 
+import markdown
+
 class ResourceEstimatorResult(Result):
+    def __init__(
+        self,
+        **kwargs
+    ) -> None:
+        super().__init__(**kwargs)
+        self.summary = ResourceEstimatorResultSummary(**kwargs)
+
+    def _repr_html_(self):
+        return self._repr_html_success_() if self.success else self._repr_html_error_()
+
+    def _repr_html_success_(self):
+        html = ""
+
+        md = markdown.Markdown(extensions=['mdx_math'])
+        for group in self.data()['reportData']['groups']:
+            html += f"""
+                <details {"open" if group['alwaysVisible'] else ""}>
+                    <summary style="display:list-item">
+                        <strong>{group['title']}</strong>
+                    </summary>
+                <table>"""
+            for entry in group['entries']:
+                val = self.data()
+                for key in entry['path'].split("/"):
+                    val = val[key]
```

Comment:
I don't know what L36 does, but it looks suspicious, especially in a loop (a variable's value is overwritten with a part of itself).

Reply:
It's is recursively entering a Python dictionary based on a path, e..g.,

`D = {'a': {'b': {'c': 42}}}`

with

`path = "a/b/c"`

would evaluate to `42`.

where `val` would be `D`, then `{'b': {'c': 42}}`, then `{'c': 42}` and finally `42`.